examples/elf/tests/helloxx/Makefile

```
@@ -110,10 +110,11 @@ $(BIN3): $(OBJS3)
 	$(Q) $(LD) $(LDELFFLAGS) $(LDLIBPATH) -o $@ $(ARCHCRT0OBJ) $^ $(LDLIBS)
 
 # BIN4 is similar to BIN3 except that it uses the streams code from libstdc++
-#
+# Both BIN4 and BIN5 use exceptions
 # NOTE:  libstdc++ is not available for NuttX as of this writing
 #
 ifeq ($(CONFIG_EXAMPLES_ELF_CXX),y)
+ifeq ($(CXX_EXCEPTION),y) 
```

I wonder how your tests pass as I expected we need `ifeq ($(CONFIG_CXX_EXCEPTION),y)` here and not `ifeq ($(CXX_EXCEPTION),y)`

---

That's weird.
```suggestion
ifeq ($(CONFIG_CXX_EXCEPTION),y)
```
Would actually check...perhaps the tests don't ever enable `CONFIG_EXAMPLES_ELF_CXX` ?